#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("debug_mode", bool_t, 0, "Run in debug mode", True)
gen.add("min_blob_area", int_t, 0, "Minimum blob area to filter", 2000, 0, 200000)
gen.add("max_blob_area", int_t, 0, "Maximum blob area to filter", 90000, 0, 200000)

exit(gen.generate("mcr_blob_detection", "mcr_blob_detection", "BlobDetection"))